apps/client/src/views/cuesheet/cuesheet-table/cuesheet-table-elements/EditableImage.tsx

```diff
@@ -0,0 +1,55 @@
+import { memo } from 'react';
+import { Input } from '@chakra-ui/react';
+
+import style from './EditableImage.module.scss';
+
+interface EditableImageProps {
+  initialValue: string;
```

Comment:
This value is undefined when the event has no data for the custom filed

it leads to it showing the delete button even though there is no value